guide/onboarding/getting-to-know-your-users.md

```diff
@@ -0,0 +1,95 @@
+main_classes: -no-top-padding
+---
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/getting-to-know-your-users.svg"
+   alt-text = "Bitcoin, key and lock"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Getting to know your users
+
+When building onboarding experiences, it is important to understand and develop knowledge about your users.
+Within the Bitcoin ecosystem, there is a wide range in the level of expertise. Identifying who you are building for is crucial, as these user groups value and understand varying characteristics of coin management tools. For example, “rookies” often favor convenient, easy-to-use wallets, while well-versed bitcoin owners tend to prioritize security[^1].
+
+## Observe your potential users
+This observation could take the form of user research, user interviews and surveys aimed at a range of participants and observe what they do, how they think, and what they want. Let's consider some things we might want to ask ourselves when improving the onboarding experience for new users:
+
+* What is their perception of Bitcoin?[^2]
+* Why do they intend to use Bitcoin, investment, savings or spending?
+* Where do they prefer to store most of their value, mobile or desktop wallets?
+* Where do they experience frustration?
+* Where does the first experience take place?
+* How did they hear about my application? Word of mouth? App store search?
+* Do first-time users have specific needs or goals?
+
+Perhaps after research, you identify your users as "cypherpunks" with sound knowledge of Bitcoin, but they are new to your app. You might build an experience that preserves as much of their privacy as possible and is super customizable from the outset.
+
+Aim to gather as much information and observations as possible in order to empathize with your users.
+
+## Identify barriers and motivators they have
+You can then use this research to identify where potential problems may exist. This will then allow you to highlight opportunities for innovation.
+
+### Rookie example
+
+Let's use an example of saving for the future and see how we can understand a user with this goal. For the sake of simplicity let's focus on one persona or maybe more accurately personal story.
+
+> Joshua Akinsola is a 21-year old Nigerian studying economics in the large city of Lagos. Growing up, his family struggled to save any money, due to a lack of financial education, high unemployment and a local currency that is inflating at 15% per year. There is a lack of reliable savings technology in his country so he opts to save money for the future using bitcoin.
+
+The uncovering of this scenario could be the result of a user interview[^3], that gathers insights into their needs and frustrations. By focusing on the emotional aspect of saving using bitcoin we may identify the following themes for barriers and motivations.
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/barriers-motivators.svg"
```

Comment:
Don't quite understand using the battery as a visual for strength of barrier/motivator. Instead of a big graphic, you could have smaller icons next to the description of each barrier/motivator, and maybe each icon can represent the barrier/motivator itself?

Reply:
Spoke a bit about this at the design guide session, I've tried to remix it a bit by adding some colour and splitting the barriers and motivators into different images. Might be a little more clearer, let me know your thoughts.